README.md

```diff
@@ -17,48 +17,62 @@ Conda instructions are provided in more detail, but you may also use `pip` and `
 **Note:** Java 11 is required. Instructions for installing Java 11 for Ubuntu or macOS are provided in the [Install with Conda](#install-with-conda) section.
 
 ### Install with pip
+
 To use `pip` to install TorchServe and the model archiver:
 
-```
-pip install torch torchtext torchvision sentencepiece
+``` bash
+pip install torch torchtext torchvision sentencepiece psutil future
```

Comment:
there's a PR that changes this: https://github.com/pytorch/serve/pull/228 step. I believe @aaronmarkham is also looking into that. Basically uses a central requirements.txt which is much nicer since it keeps all the requirements in a single location.

Reply:
I think that makes sense. I wanted to address this immediately because it's tagged as a launch blocker. If #228 is merged, we can change the instructions accordingly.

Reply:
@mycpuorg We need this in as a hotfix *before* that's going to land. Today is preferable.